lib/helpers/views/cycleTimeDataProvider.spec.js

```diff
@@ -0,0 +1,179 @@
+jest.mock('../../../model')
+const cycleTimeDataProvider = require('./cycleTimeDataProvider')
+const {model} = require('../../../model')
+const {TreeItem} = require('vscode')
+const {commands} = require('../../commands')
+
+const portal = {
+  2018: {
+    January: [{
+      label: '20th - 23rd',
+      iterationNumber: 25
+    }]
+  }
+}
+
+describe('#cycleTimeDataProvider', () => {
+  afterAll(() => jest.clearAllMocks())
+
+  it('should implement getChildren', () => {
+    const provider = new cycleTimeDataProvider({})
+    expect(typeof provider.getChildren).toBe('function')
```

Comment:
@brayovsky is this test really necessary, there's a comprehensive test below that tests for a return value